Framework/DataHandling/inc/MantidDataHandling/LoadEventNexus.h

```diff
@@ -7,14 +7,21 @@
 #include "MantidAPI/IFileLoader.h"
 #include "MantidDataHandling/BankPulseTimes.h"
 #include "MantidDataObjects/EventWorkspace.h"
-#include <nexus/NeXusFile.hpp>
-#include <nexus/NeXusException.hpp>
 #include "MantidDataObjects/Events.h"
 #include "MantidAPI/WorkspaceGroup.h"
 #include "MantidGeometry/Instrument.h"
 #include "MantidGeometry/Instrument/ParameterMap.h"
 #include "MantidKernel/TimeSeriesProperty.h"
 #include "MantidDataHandling/EventWorkspaceCollection.h"
+
+#ifdef _WIN32 // fixing windows issue causing conflict between
+// winnt char and nexus char
+#undef CHAR
```

Comment:
char conflicting with nexus is defined in winnt.h and its probably the windows compiler bug as nexus char is defined in nexus namespace.  
The compiler bug is probable as it works fine if precompiled headers are enabled but fails if they are not. 
Despite that, you still need to be clever to be able to compile this piece of code without precompiled headers.